src/layout/components/Description.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+import styled from 'styled-components';
+
+const DescriptionContainer = styled('div')`
+  text-align: center;
+  margin: auto;
+  width: 80%;
+  margin: auto;
+  margin-top: 2em;
+
+  *:not(:last-child) {
+    margin-bottom: 1em;
+  }
+  ${props => props.theme.breakpoints.down('sm')} {
+    width: 90%;
+    margin-top: 1.5em;
+
+    *:not(:last-child) {
+        margin-bottom: .8em;
+      }
+  }
+`;
+
+export const Description: React.FC<{}> = () => (
+  <DescriptionContainer>
+    <Typography color="textPrimary">So you want to work in IT? Or you just curious about opportunities in the TECH world?</Typography>
+    <Typography color="textPrimary">You&apos;re in the right place!</Typography>
+    <Typography color="textPrimary">Explore the options below and decide which path works best for you.</Typography>
```

Comment:
You could add this style for `color` to body, as we want to have the same primary text color for whole app. This style is inherited by all children, so you could remove it from here than.

Reply:
I should add color to index.css or there is some way to do it with a theme? Because I am looking for some solution, but I can't find it.

Reply:
I find something like this:

```
const theme = createMuiTheme({
  typography: {
    allVariants: {
      color:  '#222222'
    },
  },
});
```

Is this a right solution?

Reply:
looks good, if it's working than I think it's fine :)

Reply:
I check and it actually doesn't work for plain jsx elements, so I add color and font-family to index.css to body and that's work just fine.